saleor/core/tasks.py

```diff
@@ -14,20 +25,21 @@ def delete_from_storage_task(path):
 
 
 @app.task
-def delete_event_payloads_task():
+def delete_event_payloads_task(expiration_date=None):
+    expiration_date = expiration_date or timezone.now() + datetime.timedelta(minutes=60)
```

Comment:
This task is executed once per day, are you sure that we have such a short expiration date for them?